#include "msteam_server_player.h"
#include "src/game_logic/game_registry.h"
#include "src/network/message_sys.h"

///<<< BEGIN WRITING YOUR CODE
///<<< END WRITING YOUR CODE

///<<<rpc begin
void ServerPlayerTeamServiceImpl::EnterScene(EntityPtr& entity,
    const ::TeamTestRequest* request,
    ::TeamTestResponse* response)
{
///<<< BEGIN WRITING YOUR CODE
///<<< END WRITING YOUR CODE
}

///<<<rpc end
